ui/src/app/app.component.html

```diff
@@ -1,8 +1,7 @@
 <mat-toolbar color="primary">
     <span>Devfile Builder</span>
```

Comment:
Those were not part of the original acceptance criteria (so can be done later), but I was wondering if we could also change the following things before moving the UI out of the experimental mode:
- changing the favicon from the Angular logo to the `odo` logo.
- changing this title from "Devfile Builder" to something like "Devfile Builder (Beta)". A basic chip near the title, like below, would be even better but that can be done later:
![image](https://github.com/redhat-developer/odo/assets/593208/5ad1429a-b0be-4d71-8514-3f49648c80e4)

Reply:
- ok for the favicon, I'll change it as part of this PR
- For the Beta info, there is a "Work In Progress" on the right side of the toolbar